### PR TITLE
/fill-db

### PR DESCRIPTION
**Pull request by alesiahlushakova**

### CODE REVIEW

**Comment by svareyko** [comment]
Please resolve conflicts before merge

File: src/main/java/by/itech/lab/supplier/domain/Customer.java [hunk 1]
```diff
@@ -40,6 +40,8 @@
     @OneToMany(mappedBy = "customer")
     private Set<Warehouse> warehouses = new HashSet<>();
     @OneToMany(mappedBy = "customer")
+    @JsonManagedReference
```
Comment (svareyko) [inline src/main/java/by/itech/lab/supplier/domain/Customer.java]: this will not work in Entity, could you explain to me why?

**Review by svareyko — CHANGES_REQUESTED** [review]

**Review by alexeyoverchenko — APPROVED** [review]

File: src/main/java/by/itech/lab/supplier/domain/Customer.java [hunk 2]
```diff
@@ -6,7 +6,7 @@
 import lombok.EqualsAndHashCode;
 import lombok.NoArgsConstructor;
 import org.hibernate.annotations.Where;
-
+import com.fasterxml.jackson.annotation.JsonManagedReference;
```
Comment (svareyko) [inline src/main/java/by/itech/lab/supplier/domain/Customer.java:9]: please remove unused import

File: src/main/java/by/itech/lab/supplier/domain/Item.java [hunk 3]
```diff
@@ -1,13 +1,15 @@
 package by.itech.lab.supplier.domain;
 
 
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
```
Comment (svareyko) [inline src/main/java/by/itech/lab/supplier/domain/Item.java]: please remove unused import

File: src/main/java/by/itech/lab/supplier/domain/Item.java [hunk 4]
```diff
@@ -1,13 +1,15 @@
...
 import lombok.AllArgsConstructor;
 import lombok.Builder;
 import lombok.Data;
 import lombok.NoArgsConstructor;
 
 import javax.persistence.Column;
 import javax.persistence.Entity;
+import javax.persistence.FetchType;
```
Comment (svareyko) [inline src/main/java/by/itech/lab/supplier/domain/Item.java]: please remove unused import

File: src/main/java/by/itech/lab/supplier/dto/UserDto.java [hunk 5]
```diff
@@ -2,14 +2,15 @@
 
 import by.itech.lab.supplier.domain.Address;
 import by.itech.lab.supplier.domain.Application;
+import by.itech.lab.supplier.domain.Customer;
```
Comment (svareyko) [inline src/main/java/by/itech/lab/supplier/dto/UserDto.java]: please remove unused import

**Review by svareyko — APPROVED** [review]

**Review by IrynaZazyba — CHANGES_REQUESTED** [review]
doesn't pass through tests .http

**Review by IrynaZazyba — APPROVED** [review]

**Pull request merged**